From 1ce0a55f93bdddb4bdf0956f094f4198f714a42c Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Mon, 29 Sep 2014 09:25:52 +0200
Subject: [PATCH 4/4] pcap-linux: don't use TPACKETV3 for memory mmapped
 capture

There has been numerous reports from users complaining about changes in behavior
of libpcap when using libpcap compiled with TPACKETV3 support. Back out and
disable it.

See: https://github.com/the-tcpdump-group/libpcap/issues/380
See: https://github.com/the-tcpdump-group/libpcap/issues/364

Resolves: #1131500
---
 pcap-linux.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pcap-linux.c b/pcap-linux.c
index 58292c3..b3a7a51 100644
--- a/pcap-linux.c
+++ b/pcap-linux.c
@@ -193,9 +193,6 @@
   * uses many ring related structs and macros */
 # ifdef TPACKET_HDRLEN
 #  define HAVE_PACKET_RING
-#  ifdef TPACKET3_HDRLEN
-#   define HAVE_TPACKET3
-#  endif /* TPACKET3_HDRLEN */
 #  ifdef TPACKET2_HDRLEN
 #   define HAVE_TPACKET2
 #  else  /* TPACKET2_HDRLEN */
-- 
1.8.3.1

